### ISSUE TITLE
Incorrect runtime dependency on babel@6

### ISSUE THREAD

**Issue by vovkasm** [react-native-community/progress-view#45]
# Bug report

## Summary

Version 1.1.1 of module has incorrect runtime dependency on babel@6 and babel-runtime@6: [package.json](https://github.com/react-native-community/progress-view/blob/v1.1.1/package.json#L45)

This module should not declare runtime dependency on babeb at all, metro bundler will correctly process module code anyway.

## Environment info

Library version: 1.1.1

## Steps to reproduce

Not applicable

## Reproducible sample code

Not applicable

**Comment by vovkasm** [comment]
Hm... does anyone interesting? Should I create PR? 🤔

**Comment by maxammann** [comment]
This is a serious issue. If you use @babel/node and then add this progress view then the babel-node script gets overwritten. A workaround is:

```
  "resolutions": {
    "@react-native-community/progress-bar-android/@react-native-community/progress-view": "1.0.3"
  }
```

**Comment by Naturalclar** [comment]
@vovkasm @maxammann Thanks for raising the issue! I've published v1.2.1 with a fix
